home.html

```diff
@@ -0,0 +1,189 @@
+                </div>
+                <div class="fSImgInfos">
+                <div class="fSLiTitle">Mohamed Sheikh ghazouani</div>
+                <div class="fSLiTitle2"> The president of the RIM will attend this event.
+                </div>
+                <div class="fSLiseparator"></div>
+                <p class="fSLiPara">Attending this event means a lot for the citizens of Tagant.
+                </p>
+                </div>
+                </li>
+                <li class="fSLi">
+                    <div class="fSImgContainer">
+                        <img src="https://summit.cckorea.org/images/base/speaker_02.png" alt="tezafutati" class="fSImg">
+                    </div>
+                    <div class="fSImgInfos">
+                    <h1 class="fSLiTitle">Merzough Moahmed</h1>
+                    <h1 class="fSLiTitle2"> First Minister of the RIM
+                    </h1>
+                    <div class="fSLiseparator"></div>
+                    <p class="fSLiPara">He will be ensuring the security of this event.
+                    </p>
+                    </div>
+                </li>
+                <li class="fSLi">
+                    <div class="fSImgContainer">
+                        <img src="https://summit.cckorea.org/images/base/speaker_03.png" alt="tezafutati" class="fSImg">
+                    </div>
+                    <div class="fSImgInfos">
+                    <h1 class="fSLiTitle">Eddy Zein</h1>
+                    <h1 class="fSLiTitle2"> The minister of the indstry
+                    </h1>
+                    <div class="fSLiseparator"></div>
+                    <p class="fSLiPara">He is from the region and he will be present at home.
+                    </p>
+                    </div>
+                </li>
+                <li class="fSLi">
+                    <div class="fSImgContainer">
+                        <img src= "https://summit.cckorea.org/images/base/speaker_04.png" alt="tezafutati" class="fSImg">
+                    </div>
+                    <div class="fSImgInfos">
+                    <h1 class="fSLiTitle">Mamadou Shakho</h1>
+                    <h2 class="fSLiTitle2"> Minister of the environement
+                    </h2>
+                    <div class="fSLiseparator"></div>
+                    <p class="fSLiPara">He is here to ensure the safety of the environment
+                    </p>
+                    </div>
+                </li>
+                <li class="fSLi">
+                    <div class="fSImgContainer">
+                        <img src="https://summit.cckorea.org/images/base/speaker_05.png" alt="tezafutati" class="fSImg">
+                    </div>
+                    <div class="fSImgInfos">
+                    <h1 class="fSLiTitle">Macky Sall</h1>
+                    <h2 class="fSLiTitle2">The president of senegal
+                    </h2>
+                    <div class="fSLiseparator"></div>
```

Comment:
- [ ] [OPTIONAL] This section is not well arranged. Kindly see to it

Yours | Original
----- | -----
![image](https://user-images.githubusercontent.com/16058470/133788222-af3ef814-f99c-4a2f-83fc-2aa81f7da367.png) | ![image](https://user-images.githubusercontent.com/16058470/133788306-ff11c229-43da-4373-9805-3de1404725e4.png)